components/calendar/css/_mixin.scss

```diff
@@ -10,6 +10,9 @@
 }
 
 @mixin utrecht-calendar__table-weeks-item {
+  line-height: var(--utrecht-calendar-table-weeks-item-line-height, inherit);
```

Comment:
Could effectively do the same without editing the line height. Editing lineheight just makes it easier to stay in line with the design from Figma, and I imagine that also goes for other organisations so this token would be useful. (Den Haag design for example has a lot of heights based on specific padding + specific line height)